src/theme/DocItem/Footer/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from "react";
```

Comment:
```suggestion
// Why is this swizzled?
//   To add a PushFeedback widget to every doc, to solicit feedback from readers.
// Swizzled from version 2.4.1.

import React, { useEffect } from "react";
```

It might seem obvious within this PR, but if we make more changes to this swizzled component in the future, it's nice to have a log of why (and when) we swizzled.